.github/workflows/basic.yml

```diff
@@ -208,11 +210,28 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Get CMake for lukka/run-vcpkg
+      uses: lukka/get-cmake@latest
```

Comment:
hmm... is this really a requirement for run-vcpkg action? I can't see it listed on their GH page

Reply:
I can't see it listed on their GH page either. @KFilipek  is this really a requirement for run-vcpkg action?

It installs the latest CMake and Ninja. Let's try remove it.

Reply:
Removed

Reply:
According to: https://github.com/lukka/run-vcpkg#use-vcpkgs-vcpkgjson-file-to-specify-the-dependencies
Other way to install:
Or it can also be used by invoking `vcpkg install` in a directory where vcpkg.json is located (e.g., input runVcpkgInstall : true).